walk-in-interview/src/main/webapp/script.js

```diff
@@ -45,12 +46,18 @@ function renderHomepageElements() {
   const newPostButton = document.getElementById('new-post');
   newPostButton.innerText = STRINGS['new-post'];
   newPostButton.addEventListener('click', (_) => {
-    window.location.href= JOBPAGE_PATH;
+    window.location.href = JOBPAGE_PATH;
   });
 
   const accountButton = document.getElementById('account');
   accountButton.innerText = STRINGS['account'];
 
+  const loginButton = document.getElementById('log-in');
+  loginButton.innerText = STRINGS['log-in'];
+  loginButton.addEventListener('click', (_) => {
```

Comment:
I think you can just add this event listener to the accountButton, and then there's no need for the loginButton